components/brave_rewards/resources/tip/components/tipSite.tsx

```diff
@@ -23,7 +24,12 @@ class TipSite extends React.Component<Props, {}> {
   }
 
   onTweet = () => {
-    this.actions.onTweet(this.props.publisher.name, '')
+    let name = this.props.publisher.name
+    if (this.props.publisher.provider === 'twitter') {
```

Comment:
please check if url is null/undefined and that length is > 0

Reply:
Fixed.